src/pages/storage/forms/StorageVolumeFormMain.tsx

```diff
@@ -45,9 +32,26 @@ const StorageVolumeFormMain: FC<Props> = ({ formik, project }) => {
             help={
               formik.values.isCreating
                 ? undefined
-                : "Click the name in the header to rename the volume"
+                : "Click the name in the header to rename the volume."
             }
           />
+
+          <Label forId="storage-pool-selector" required>
+            Storage pool
+          </Label>
+          <StoragePoolSelector
+            project={project}
+            value={formik.values.pool}
+            setValue={(val) => void formik.setFieldValue("pool", val)}
+            hidePoolsWithUnsupportedDrivers
+            disabled={!formik.values.isCreating}
+            help={
+              formik.values.isCreating
+                ? undefined
+                : "Click 'Migrate' in the actions list to move the volume to a different storage pool."
```

Comment:
```suggestion
                : "Use the migrate button in the header to move the volume to a different storage pool."
```
Maybe better? Users might not know what "action list" this refers to.